modules/govuk/manifests/apps/collections.pp

```diff
@@ -41,6 +45,7 @@
     vhost                  => $vhost,
     nagios_memory_warning  => $nagios_memory_warning,
     nagios_memory_critical => $nagios_memory_critical,
+    sentry_dsn             => $sentry_dsn,
```

Comment:
Given that this puppet thing uses `govuk::app::envvar`, I think it would be easier to understand if `govuk::app` wasn't used here, as all it is doing is passing this value through to `govuk:app:envvar`.

This would also match up with how Errbit is configured at the moment.